devicetypes/krlaframboise/sensative-strips-comfort.src/sensative-strips-comfort.groovy

```diff
@@ -0,0 +1,774 @@
+/*
```

Comment:
Since **zwave-water-temp-light-sensor.groovy** already supports this device - if there are any changes needed, please add them in this DTH:
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zwave-water-temp-light-sensor.src/zwave-water-temp-light-sensor.groovy

Reply:
Hello Konrad,

We had received the following feedback from SmartThings in the past,

"We are in the middle of certifying Comfort and Drip and ran into an unusual situation.  It seems that the devices share a Z-Wave certification; meaning that when we pair the devices, we have no way of differentiating between the two products.  Since they share the same code and capabilities, they share the same Device Type Handler.  Therefore, the SmartThings platform see's them as the same device."

Also, based on the valuable feedback we have received from the SmartThings end users in the past couple of years, and after consulting with SmartThings, we decided to outsource the development of SmartThings DTH that could differentiate the two products Strips Comfort and Strips Drip and also maximize the user experience of these products towards the SmartThings ecosystem. Our test team has verified these device handlers thoroughly.

Therefore, we request you to kindly review these new device handlers and adapt the same. Let us know if we need to update any changes within these new device handlers.

Best Regards,
Dhiraj Paryani

Reply:
@dparyani Please take a look at my last comment in #50493